guacamole-server.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: guacamole-server
+  version: 1.5.2
+  epoch: 0
+  description: clientless remote desktop gateway server
+  copyright:
+    - license: GPL-3.0-or-later
```

Comment:
```suggestion
    - license: Apache-2.0
```

Reply:
Hmm, I grabbed it from here: https://git.alpinelinux.org/aports/tree/testing/guacamole-server/APKBUILD

but it does look like it's apache